client/src/graphs/templates/templates.test.ts

```diff
@@ -0,0 +1,91 @@
+import { describe, it, expect } from "vitest";
+import {
+  getAverageCoordinatesOfGraphNodes,
+  centerNodesOnOriginCoordinates,
+} from "./helpers";
+
+describe("getAverageCoordinatesOfGraphNodes", () => {
+  it("should return the average coordinates for a list of nodes", () => {
+    const nodes = [
+      { x: 10, y: 20 },
+      { x: 30, y: 40 },
+      { x: 50, y: 60 },
+    ];
+
+    const result = getAverageCoordinatesOfGraphNodes(nodes);
+    expect(result).toEqual({ x: 30, y: 40 });
+  });
+
+  it("should handle a single node correctly", () => {
+    const nodes = [{ x: 15, y: 25 }];
+
+    const result = getAverageCoordinatesOfGraphNodes(nodes);
+    expect(result).toEqual({ x: 15, y: 25 });
+  });
+
+  it("should return { x: 0, y: 0 } for an empty node list", () => {
```

Comment:
handles edge case of 0 nodes instead of specifically putting the object inside the name